backend/slack_commands/execution/session_executor.py

```
@@ -0,0 +1,164 @@
+
+    @staticmethod
+    def _extract_error_body(error: requests.HTTPError) -> str:
+        try:
+            if error.response is not None:
+                return error.response.json().get("error", "")
+        except Exception:
+            pass
+        return ""
+
+    @staticmethod
+    def _post_to_slack(response_url: str, text: str, success: bool = False) -> None:
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)